next.config.js

```diff
@@ -9,11 +9,7 @@ const nextConfig = {
       's3-us-gov-west-1.amazonaws.com',
       'va.gov',
     ],
-  },
-  experimental: {
-    images: {
-      unoptimized: true,
-    },
+    loader: 'custom',
```

Comment:
@amponce I'd like your review in particular to see if what I've done to image handling in this PR matches with the original intention for how we're handling images in here. We can't rely on Next/Vercel image optimization for export. Export fails if it's in place, because at export time there's not a Next server running. The way we had set it up previously was triggering that complaint.

Reply:
I'm going to pull this repo down and take a look at this right now

Reply:
So I think we can remove the unoptimized prop from the Image component and the failing test can be updated to

```
 test('throws an error if no URL is passed in', () => {
    // eslint-disable-next-line
    expect(() => render(<Image src={url} alt=''/>)).toThrowError(
      /url is not defined/i
    )
  })
  
  ```

Reply:
I just did a trial run and the export didn't work with the images, I'll see if there's another way to do it